content/learning-paths/servers-and-cloud-computing/cca-kata/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Run Confidentail Containers with encrypted images using Arm CCA and Trustee
+
+draft: true
+cascade:
+    draft: true
+    
+minutes_to_complete: 60
+
+who_is_this_for: This Learning Path is for software developers who want to understand how Confidential Containers can be run in Arm CCA realm.
+
+learning_objectives:
+  - Overview of Confidential Containers
+  - Understand how Trustee services are used for CCA realm attestation to unlock the confidential processing of data.
```

Comment:
Nitpick: "CCA realm attestation" should either be expanded to "CCA realm and platform attestation" or contracted to just "CCA attestation" (rather than specifically saying "realm", due to the way that CCA attestation is partitioned into realm/platform, and we need to consider both).